manual.docbook

```diff
@@ -1158,248 +1264,125 @@
-            one of the arrow keys.
+            Using the arrow keys on the keyboard, and the <keycap>Return</keycap>,
+            will also add and remove patterns from the song.
           </para>
+
+          <note>
+	        <para>
+              The keyboard input cursor is usually hidden unless you press one
+              of the keys listed in <xref linkend="chpt.keyboard_and_mouse" />.
+              You can alter this default behavior in the <link
+              linkend="chpt.preferences.general_tab">General tab</link> of the
+              Preferences.
+            </para>
+          </note>
+          
         </listitem>
 
         <listitem>
           <para><inlinemediaobject><imageobject>
             <imagedata fileref="img/btn_select.png" format="PNG"/>
-            </imageobject></inlinemediaobject> Enable Select Mode.  This mode allows 
-            you to select a part of the song and delete/move/copy it.</para>
+            </imageobject></inlinemediaobject> : enables Select Mode.  This mode allows 
+            you to select multiple patterns in the Song Editor and delete/move/copy them.</para>
 		<para>Once you have selected a part of your song you can <emphasis role="bold">delete</emphasis> 
-			it by pressing the Delete button.  You can <emphasis
+			it by pressing the <keycap>Delete</keycap>.  You can <emphasis
              role="bold">move</emphasis> it by simply dragging your selection
-            to another location, and you can also <emphasis role="bold">copy</emphasis>
-             your selection by Ctrl-dragging it to a new location.
+            to another location with your mouse or by cutting (<keycap>Ctrl</keycap> + <keycap>x</keycap>) and pasting (<keycap>Ctrl</keycap> + <keycap>v</keycap>) using your kyeboard. You can also <emphasis role="bold">copy</emphasis>
+             your selection by either holding <keycap>Ctrl</keycap> while dragging it to a new location or by copying (<keycap>Ctrl</keycap> + <keycap>c</keycap>) and pasting (<keycap>Ctrl</keycap> + <keycap>v</keycap>) using your kyeboard.
           </para>
           <para>
-            Selections can be modified by Ctrl-clicking to select
+            Selections can be modified by holding <keycap>Ctrl</keycap> while clicking to select
             additional blocks, or to remove selected blocks from the
             selection.
           </para>
           <para>
             The arrow keys on the keyboard can also be used, along
-            with the Return key, to <emphasis
+            with <keycap>Return</keycap>, to <emphasis
             role="bold">select</emphasis>, <emphasis
             role="bold">move</emphasis> and <emphasis
-            role="bold">copy</emphasis> (Ctrl+Return) parts of the
+            role="bold">copy</emphasis> (<keycap>Ctrl</keycap> + <keycap>Return</keycap>) parts of the
             song.
+
+            TODO: is this still valid? It doesn't work for me.
```

Comment:
It works for me in macOS and Linux. Maybe the sequence isn't quite clear, I'll expand that:
  - <kbd>Return</kbd> over a selected block starts the move/copy
  - Arrow keys to move position
  - <kbd>Return</kbd> to move the blocks into place, or
  - <kbd>Ctrl</kbd> + <kbd>Return</kbd> to copy blocks to the new location.
 Does that work for yoU?

Reply:
Ah, I see. Yes, it works for me too. We should probably put this description as <procedure><step>.. into the manual.

Reply:
WilCo